pkg/blocktx/client.go

```diff
@@ -15,6 +15,18 @@ type BlocktxClient interface {
 	ClearBlocks(ctx context.Context, retentionDays int32) (int64, error)
 	ClearBlockTransactionsMap(ctx context.Context, retentionDays int32) (int64, error)
 	DelUnfinishedBlockProcessing(ctx context.Context, processedBy string) error
+	VerifyMerkleRoots(ctx context.Context, merkleRootVerificationRequest []MerkleRootVerificationRequest) ([]uint64, error)
```

Comment:
`BlocktxClient` interface doesn't need `VerifyMerkleRoots` method

Reply:
Unfortunately it does, because the method `blocktx.NewClient()` returns this interface and we want to pass blocktx as `MerkleRootsVerificator` into API handler, for example in `/cmd/arc/services/api.go` line `126`. So it needs to implement the method `VerifyMerkleRoots` from `MerkleRootsVerificator` interface.

Reply:
ehh, who needs interface segregation...